src/bundle/Resources/translations/section.en.xliff

```diff
@@ -112,8 +112,8 @@
         <note>key: section.name</note>
       </trans-unit>
       <trans-unit id="12023a25e9c04ceef708820416f14a5a1e188688" resname="section.new">
-        <source>Create a new Section</source>
-        <target state="new">Create a new Section</target>
+        <source>Create</source>
```

Comment:
Should't we have "Create section"?

Reply:
In the ticket it says "Create", I think it is now unified with the rest of the system.

Reply:
OK